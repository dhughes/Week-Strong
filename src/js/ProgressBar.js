import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  flex: 1;
  display: flex;
  background-color:  ${props => props.theme.font};
  height: 2em;
`;

const Progress = styled.div`
  background-color:  ${props => props.theme.header};
  width: ${props => props.progress}%;
  min-height: 2em;
  content: '.';
  display: hidden;
`;

const ProgressBar = () => {
  return <Container><Progress progress={75} /></Container>;
};

export default ProgressBar;
