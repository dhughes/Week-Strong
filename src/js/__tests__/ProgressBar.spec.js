import React from 'react';
import { shallow } from 'enzyme';
import ProgressBar from '../ProgressBar';

test('ProgressBar renders', () => {
  const component = shallow(<ProgressBar />);
  expect(component).toMatchSnapshot();
});
