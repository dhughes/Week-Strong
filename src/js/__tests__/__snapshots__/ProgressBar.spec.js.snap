// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar renders 1`] = `
<styled.div>
  <styled.div
    progress={75}
  />
</styled.div>
`;
